app/views/submissions/_react_file_manager_boot_table.js.jsx.erb

```diff
@@ -59,7 +59,8 @@
       this.setState({ selected_files: [] });
 
       jQuery.ajax({
-        url: '<%= populate_file_manager_react_assignment_submissions_path %>',
+        url: '<%= populate_file_manager_react_assignment_submissions_path(
+          path: @path) %>',
```

Comment:
I get what you were doing here, but it's more standard to pass this as part of the `data` option in the ajax call. Something like

```javascript
method: 'GET',
data: { path: '<%= @path %>' },
...
```